plugins/system/net.go

```diff
@@ -79,6 +80,15 @@ func (s *NetIOStats) Gather(acc plugins.Accumulator) error {
 		acc.Add("drop_out", io.Dropout, tags)
 	}
 
+	// Get system wide stats for different network protocols
+	netprotos, err := s.ps.NetProto()
```

Comment:
If we're ignoring the error just set the var name to `_` to make it more clear that we don't care if this call fails